testing/it/e2e-selenium/src/test/java/com/adobe/cq/wcm/core/components/it/seljup/components/embed/UrlProcessors.java

```diff
@@ -0,0 +1,85 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2021 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+package com.adobe.cq.wcm.core.components.it.seljup.components.embed;
+
+import com.adobe.cq.testing.selenium.pagewidgets.Helpers;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class UrlProcessors {
+    public class OEmbed {
+        private String name;
+        private String selector;
+        private String [] urls;
+
+        public String getName() {
+            return name;
+        }
+
+        public boolean isUrlProcessorExits() {
```

Comment:
Is this method checking for existence of URL processor? If so, I'd rename it `urlProcessorExists`.